cache/contenthash/checksum.go

```diff
@@ -610,7 +610,7 @@ func (cc *cacheContext) checksum(ctx context.Context, root *iradix.Node, txn *ir
 		return nil, false, err
 	}
 	if cr == nil {
-		return nil, false, errors.Wrapf(errNotFound, "%q not found", convertKeyToPath(origk))
+		return nil, false, errors.Wrapf(errNotFound, "%q", convertKeyToPath(origk))
```

Comment:
I kept the `%q` formatting here, but perhaps it's not needed, and could be changed to

```suggestion
		return nil, false, errors.Wrap(errNotFound, convertKeyToPath(origk))
```

Reply:
Actually; I think the `%q` could be useful; looking at the original error;

    failed to compute cache key: "/no-such-file.txt" not found: not found

Which would now become

    failed to compute cache key: "/no-such-file.txt": not found

Removing the quotes would make it;

    failed to compute cache key: /no-such-file.txt: not found

Which "works" for simple cases, but possibly in other cases, e.g. empty  filename, or path with spaces, would be not so clear.